--- /cygdrive/c/Documents and Settings/Matthew/My Documents/Projects/TheyWorkForYou/NI/parldata/cmpages/ni/ni1998-07-01.html	2006-11-12 10:26:15.282238400 +0000
+++ /cygdrive/c/Documents and Settings/Matthew/My Documents/Projects/TheyWorkForYou/NI/parldata/tmp/patchtmp-ni1998-07-01-Kwno1o.html	2006-12-05 16:06:19.086179200 +0000
@@ -65,7 +65,7 @@
                 <br>
                 </p>
                 </font></i><font SIZE="3"><b>
-                <p ALIGN="JUSTIFY">The Initial Presiding Officer:</p>
+                <p ALIGN="JUSTIFY">The Initial Presiding Officer (Lord Alderdice):</p>
                 </b>
                 <p ALIGN="JUSTIFY">Welcome to the first meeting of the New
                 Northern&nbsp;Ireland Assembly.</p>
@@ -2384,7 +2384,7 @@
                 Member has any objection to the three Independents’ having a
                 representative. If not, is there any difficulty?</p>
                 <b>
-                <p ALIGN="JUSTIFY">The</b></font><b><font SIZE="2"> </font><font SIZE="3">Initial
+                <p ALIGN="JUSTIFY">The Initial
                 Presiding Officer:</p>
                 </font></b><font SIZE="3">
                 <p ALIGN="JUSTIFY">Yes, there is a difficulty. With regard to
@@ -2636,8 +2636,8 @@
                 that were the wish of the Assembly, which she says she wants to
                 bolster.</p>
                 <b>
-                <p ALIGN="JUSTIFY">The</b></font><b><font SIZE="2"> </font><font SIZE="3">Initial
-                Presiding</font><font SIZE="2"> </font><font SIZE="3">Officer:</p>
+                <p ALIGN="JUSTIFY">The Initial
+                Presiding Officer:</p>
                 </font></b><font SIZE="3">
                 <p ALIGN="JUSTIFY">I will take Mr&nbsp;McCartney’s point of
                 order and then reply to both.</p>
@@ -2657,7 +2657,7 @@
                 how could the hour’s notice which, apparently, these Standing
                 Orders require have been given?</p>
                 <b>
-                <p ALIGN="JUSTIFY">The Initial Presiding Order:</p>
+                <p ALIGN="JUSTIFY">The Initial Presiding Officer:</p>
                 </b>
                 <p ALIGN="JUSTIFY">I am grateful to Members for pointing out the
                 difficulty in which I and some others were put. Several meetings
