--- /cygdrive/c/Documents and Settings/Matthew/My Documents/Projects/TheyWorkForYou/NI/parldata/cmpages/ni/ni1999-11-29.html	2006-11-12 10:26:20.760115200 +0000
+++ /cygdrive/c/Documents and Settings/Matthew/My Documents/Projects/TheyWorkForYou/NI/parldata/tmp/patchtmp-ni1999-11-29-i-xbve.html	2006-12-05 17:27:53.403862400 +0000
@@ -2609,7 +2609,7 @@
                 <p>I cannot sit through this obscenity. I am leaving.</p>
                 <p><a href="#top">TOP</a></p>
                 <b>
-                <p>The</b></font><b><font SIZE="2"> </font><font SIZE="3">Initial
+                <p>The Initial
                 Presiding Officer:</p>
                 </font></b><font SIZE="3">
                 <p>Order. Will Mr McGuinness confirm that he is willing to take
@@ -3118,7 +3118,7 @@
                 <p>The sitting was suspended at 7.42&nbsp;pm and resumed at
                 7.52&nbsp;pm.</p>
                 </i><b>
-                <p>The </b></font><b><font SIZE="3">Initial Presiding Officer:</p>
+                <p>The Initial Presiding Officer:</p>
                 </font></b><font SIZE="3">
                 <p>I call Mr&nbsp;Mitchel McLaughlin to make a nomination.</p>
                 <b>
