--- /cygdrive/c/Documents and Settings/Matthew/My Documents/Projects/TheyWorkForYou/NI/parldata/cmpages/ni/ni1999-02-01.html	2006-11-12 10:26:09.393771200 +0000
+++ /cygdrive/c/Documents and Settings/Matthew/My Documents/Projects/TheyWorkForYou/NI/parldata/tmp/patchtmp-ni1999-02-01-9ldXuC.html	2006-12-05 16:55:01.067782400 +0000
@@ -218,7 +218,7 @@
                 <p ALIGN="JUSTIFY">Do you not accept that on all these matters
                 ‘Erskine&nbsp;May’ is not specific but general?</p>
                 <b>
-                <p ALIGN="JUSTIFY">The Initial Officer:</p>
+                <p ALIGN="JUSTIFY">The Initial Presiding Officer:</p>
                 </b>
                 <p ALIGN="JUSTIFY">The Member must consult ‘Erskine&nbsp;May’.
                 On this matter it is not general but highly specific.</p>
@@ -687,7 +687,7 @@
                 <p ALIGN="JUSTIFY">Obviously, some people are not expert in the
                 Standing Order. This is a 20-minute speech. <i>[Interruption]</p>
                 </i><b>
-                <p ALIGN="JUSTIFY">The Temporary Chairperson:</p>
+                <p ALIGN="JUSTIFY">The Temporary Chairperson (Ms Morrice):</p>
                 </b>
                 <p ALIGN="JUSTIFY">Order.</p>
                 <b>
