--- /cygdrive/c/Documents and Settings/Matthew/My Documents/Projects/TheyWorkForYou/NI/parldata/cmpages/ni/ni1999-12-06.html	2006-11-12 10:26:25.727257600 +0000
+++ /cygdrive/c/Documents and Settings/Matthew/My Documents/Projects/TheyWorkForYou/NI/parldata/tmp/patchtmp-ni1999-12-06-Mo_9of.html	2006-12-05 17:29:31.755284800 +0000
@@ -276,8 +276,7 @@
                 which we possess by virtue of our being human — is a matter of
                 philosophical dispute. However, what is not a matter of dispute,
                 short of denying that there are any human rights, is that the
-                most fundamental human right is the right to life<b>.</p>
-                </b>
+                most fundamental human right is the right to life.</p>
                 <p ALIGN="JUSTIFY">The right to life is fundamental in the
                 crucial sense that all other human rights are derived from it.
                 The most immediate human right derived from the right to life is
