--- /cygdrive/c/Documents and Settings/Matthew/My Documents/Projects/TheyWorkForYou/NI/parldata/cmpages/ni/ni1999-03-09.html	2006-11-12 10:26:16.694268800 +0000
+++ /cygdrive/c/Documents and Settings/Matthew/My Documents/Projects/TheyWorkForYou/NI/parldata/tmp/patchtmp-ni1999-03-09-7xdTYK.html	2006-12-06 10:38:10.293662400 +0000
@@ -235,19 +235,18 @@
                 <p ALIGN="JUSTIFY">I have no requests from Members to speak on
                 this group of amendments, so we shall simply take decisions on
                 them and on the relevant Standing Orders.</p>
-                <b>
                 <p ALIGN="JUSTIFY">Standing Order 27 (Public Bills: Introduction
-                and First Stage)<i> </i></b><i>agreed to.</p>
-                </i><b>
+                and First Stage)<i> agreed to.</p>
+                </i>
                 <p ALIGN="JUSTIFY">Standing Order 28 (Stages in Consideration of
-                Public Bills) </b><i>agreed to.</p>
-                </i><b>
+                Public Bills) <i>agreed to.</p>
+                </i>
                 <p ALIGN="JUSTIFY">Standing Order 29 (Public Bills: Second
-                Stage)</b> <i>agreed to.</p>
-                </i><b>
+                Stage) <i>agreed to.</p>
+                </i>
                 <p ALIGN="JUSTIFY">Standing Order 30 (Public Bills: Committee
                 Stage)</p>
-                </b><i>
+                <i>
                 <p ALIGN="JUSTIFY">Amendment (No&nbsp;22) made: </i>In paragraph
                 (3) leave out &quot;to be made to the Bill&quot; and insert
                 &quot;, to the Bill, that may be proposed&quot;. —<i>
@@ -256,13 +255,13 @@
                 (5), line 3, leave out &quot;in&quot; and insert &quot;on&quot;.
                 —<i> [Mr&nbsp;P&nbsp;Robinson]</p>
                 <p ALIGN="JUSTIFY">Standing Order 30, as amended, agreed to.</p>
-                </i><b>
+                </i>
                 <p ALIGN="JUSTIFY">Standing Order 31 (Public Bills: Human Rights
-                Issues)</b> <i>agreed to.</p>
-                </i><b>
+                Issues) <i>agreed to.</p>
+                </i>
                 <p ALIGN="JUSTIFY">Standing Order 32 (Public Bills: Equality
                 Issues)</p>
-                </b><i>
+                <i>
                 <p ALIGN="JUSTIFY">Amendment (No&nbsp;20) made: </i>In paragraph
                 (1) leave out &quot;human rights&quot; and insert &quot;equality
                 requirements&quot;. —<i>&nbsp;[Mr&nbsp;P&nbsp;Robinson]</p>
@@ -278,38 +277,38 @@
                 <i>
                 <p ALIGN="JUSTIFY">Standing Order 32, as amended, agreed to.</p>
                 <p ALIGN="JUSTIFY">10.45 am</p>
-                </i><b>
+                </i>
                 <p ALIGN="JUSTIFY">Standing Order 33 (Public Bills:
-                Consideration Stage)</b> <i>agreed to.</p>
-                </i><b>
+                Consideration Stage) <i>agreed to.</p>
+                </i>
                 <p ALIGN="JUSTIFY">Standing Order 34 (Public Bills: Amendments)</p>
-                </b><i>
+                <i>
                 <p ALIGN="JUSTIFY">Amendment (No&nbsp;16) made: </i>Leave out
                 &quot;bill&quot; and insert &quot;Bill&quot;. —<i>
                 [Mr&nbsp;P&nbsp;Robinson]</p>
                 <p ALIGN="JUSTIFY">Standing Order 34, as amended, agreed to.</p>
-                </i><b>
+                </i>
                 <p ALIGN="JUSTIFY">Standing Order 35 (Public Bills: Final Stage)</p>
-                </b><i>
+                <i>
                 <p ALIGN="JUSTIFY">Amendment (No&nbsp;15) made: </i>In
                 paragraph&nbsp;(1) leave out &quot;in&quot; and insert
                 &quot;on&quot;. —<i> [Mr&nbsp;P&nbsp;Robinson]</p>
                 <p ALIGN="JUSTIFY">Standing Order 35, as amended, agreed to.</p>
-                </i><b>
+                </i>
                 <p ALIGN="JUSTIFY">Standing Order 36 (Public Bills:
-                Reconsideration) </b><i>agreed to.</p>
-                </i><b>
+                Reconsideration) <i>agreed to.</p>
+                </i>
                 <p ALIGN="JUSTIFY">Standing Order 37 (Public Bills: Explanatory
                 and Financial Memoranda)</p>
-                </b><i>
+                <i>
                 <p ALIGN="JUSTIFY">Amendment (No&nbsp;14) made: </i>In
                 paragraph&nbsp;(c), after &quot;the&quot;, insert
                 &quot;main&quot;. —<i> [Mr&nbsp;P&nbsp;Robinson]</p>
                 <p ALIGN="JUSTIFY">Standing Order 37, as amended, agreed to.</p>
-                </i><b>
+                </i>
                 <p ALIGN="JUSTIFY">Standing Order 38 (Public Bills: Special
                 Scheduling Requirement)</p>
-                </b><i>
+                <i>
                 <p ALIGN="JUSTIFY">Amendment (No&nbsp;13) made: </i>In
                 paragraph&nbsp;(2)(a), after &quot;the&quot; insert &quot;reason
                 or&quot;. —<i> [Mr&nbsp;P&nbsp;Robinson]</p>
@@ -317,10 +316,10 @@
                 paragraph&nbsp;(2), after &quot;he&quot; insert
                 &quot;/she&quot;.<i> — [Mr P Robinson]</p>
                 <p ALIGN="JUSTIFY">Standing Order 38, as amended, agreed to.</p>
-                </i><b>
+                </i>
                 <p ALIGN="JUSTIFY">Standing Order 39 (Statutory Rules of
                 Northern&nbsp;Ireland)</p>
-                </b><i>
+                <i>
                 <p ALIGN="JUSTIFY">Amendment (No&nbsp;11) made: </i>In
                 paragraph&nbsp;(3), after &quot;require&quot;, insert &quot;the
                 Minister responsible for&quot;. —<i>&nbsp;[Mr&nbsp;P&nbsp;Robinson]</p>
@@ -452,10 +451,9 @@
                 </b>
                 <p ALIGN="JUSTIFY">As I have no further requests to speak, I
                 propose that we proceed to the voting.</p>
-                <b>
                 <p ALIGN="JUSTIFY">Standing Order 40 (Appointment of Northern
                 Ireland Ministers — Time Limits)</p>
-                </b><i>
+                <i>
                 <p ALIGN="JUSTIFY">Amendment (No 45) made: </i>In paragraph (1)
                 leave out &quot;follows&quot; and insert &quot;set out in
                 Standing Order 40(2), (2a) and (3)&quot;. —&nbsp;[Mr&nbsp;P&nbsp;Robinson]</p>
@@ -545,9 +543,9 @@
                 paragraph&nbsp;(3)(b), after &quot;gives&quot;, insert &quot;a
                 reason or&quot;. —<i> [Mr&nbsp;P&nbsp;Robinson]</p>
                 <p ALIGN="JUSTIFY">Standing Order 40, as amended, agreed to.</p>
-                </i><b>
+                </i>
                 <p ALIGN="JUSTIFY">Standing Order 41 (Junior Ministers)</p>
-                </b><i>
+                <i>
                 <p ALIGN="JUSTIFY">Amendment (No 44) made: </i>Leave out
                 &quot;(1)&quot;. —<i>&nbsp;[Mr&nbsp;P&nbsp;Robinson]</p>
                 <p ALIGN="JUSTIFY">Amendment (No 61) made:</i> In line 3, leave
@@ -559,7 +557,7 @@
                 </b></i>
                 <p ALIGN="JUSTIFY"><a href="#top">TOP</a></p>
                 <b>
-                <p ALIGN="JUSTIFY">The Initial Presiding Office:</p>
+                <p ALIGN="JUSTIFY">The Initial Presiding Officer:</p>
                 </b>
                 <p ALIGN="JUSTIFY">We now come to the section on Committees.</p>
                 <b>
@@ -2734,15 +2732,15 @@
                 <p ALIGN="JUSTIFY">We come now to the approval of the Standing
                 Orders and amendments in the first group, which relates to
                 committees.</p>
-                <b>
                 <p ALIGN="JUSTIFY">Standing Order 42 (Committees of the
-                Assembly: General)</b> <i>agreed to.</p>
-                </i><b>
-                <p ALIGN="JUSTIFY">Standing Order 43 (Statutory Committees)</b> <i>agreed
+                Assembly: General) <i>agreed to.</p>
+                </i>
+                <p ALIGN="JUSTIFY">Standing Order 43 (Statutory Committees) <i>agreed
                 to.</p>
-                </i><b>
+                </i>
                 <p ALIGN="JUSTIFY">Standing Order 44 (Establishment of Statutory
                 Committees)</p>
+		<b>
                 <p ALIGN="JUSTIFY">The Initial Presiding Officer:</p>
                 </b>
                 <p ALIGN="JUSTIFY">Is amendment No&nbsp;60 moved or not moved?</p>
@@ -4743,19 +4741,18 @@
                 remind Members of the situation in respect of
                 amendments&nbsp;No&nbsp;3A to E and 4, 7, 8, 9 and&nbsp;10 when
                 we come to consider them.</p>
-                <b>
                 <p ALIGN="JUSTIFY">Standing Order 45 (Membership of Statutory
                 Committees)</p>
-                </b><i>
+                <i>
                 <p ALIGN="JUSTIFY">Amendment&nbsp;(No&nbsp;62) made: </i>In
                 paragraph (5) leave out &quot;Allocation&quot; and insert
                 &quot;allocation&quot;. —&nbsp;[Mr&nbsp;P&nbsp;Robinson]</p>
                 <i>
                 <p ALIGN="JUSTIFY">Standing Order 45, as amended, agreed to.</p>
-                </i><b>
+                </i>
                 <p ALIGN="JUSTIFY">Standing Order 46 (Overlap of Statutory
                 Committee Business)</p>
-                </b><i>
+                <i>
                 <p ALIGN="JUSTIFY">Amendment (No&nbsp;53) made: </i>In paragraph
                 (2), at end, add</p>
                 </font><font SIZE="1">
@@ -4764,16 +4761,16 @@
                 with a draft report&quot;. —<i>&nbsp;[Mr&nbsp;P&nbsp;Robinson]</p>
                 </i></font><font SIZE="3"><i>
                 <p ALIGN="JUSTIFY">Standing Order 46, as amended, agreed to.</p>
-                </i><b>
-                <p ALIGN="JUSTIFY">Standing Order 47 (Non-Statutory Committees) </b><i>agreed
+                </i>
+                <p ALIGN="JUSTIFY">Standing Order 47 (Non-Statutory Committees) <i>agreed
                 to.</p>
-                </i><b>
-                <p ALIGN="JUSTIFY">Standing Order 48 (Service on Committees) </b><i>agreed
+                </i>
+                <p ALIGN="JUSTIFY">Standing Order 48 (Service on Committees) <i>agreed
                 to.</p>
-                </i><b>
+                </i>
                 <p ALIGN="JUSTIFY">Standing Order 49 (Committee Members Voting
                 in the Chamber)</p>
-                </b><i>
+                <i>
                 <p ALIGN="JUSTIFY">Amendment (No 52) made: </i>After the first
                 &quot;shall&quot; leave out</p>
                 </font><font SIZE="1">
@@ -4782,9 +4779,9 @@
                 </i></font><font SIZE="3"><i>
                 <p ALIGN="JUSTIFY">Standing Order&nbsp;49, as amended, agreed
                 to.</p>
-                </i><b>
+                </i>
                 <p ALIGN="JUSTIFY">Standing Order 50 (Sub-Committees)</p>
-                </b><i>
+                <i>
                 <p ALIGN="JUSTIFY">Amendment (No 51) made: </i>At end add</p>
                 </font><font SIZE="1">
                 <p ALIGN="JUSTIFY">&quot;Such Committees shall, in as far as is
@@ -4793,11 +4790,12 @@
                 </i></font><font SIZE="3"><i>
                 <p ALIGN="JUSTIFY">Standing Order&nbsp;50, as amended, agreed
                 to.</p>
-                </i><b>
+                </i>
                 <p ALIGN="JUSTIFY">Standing Orders 51 (Committee on Procedures);
                 52 (Business Committee); 53 (Conformity with Equality
                 Requirements — Special Committee on); 54 (Public Accounts
                 Committee); and 55 (Committee on Standards and Privileges)</p>
+		<b>
                 <p ALIGN="JUSTIFY">The Initial Presiding Officer:</p>
                 </b>
                 <p ALIGN="JUSTIFY">We now come to amendment 3A, with which we
@@ -4878,11 +4876,11 @@
                 33&nbsp;Nationalists and 23 Unionists, and the Noes were 28
                 Unionists and 7 Others.</p>
                 <p ALIGN="JUSTIFY">Amendments Nos 4, 7, 8, 9 and 10 not moved.</p>
-                <b>
-                <p ALIGN="JUSTIFY">Standing Order 51,<i> </i></b><i>as amended
+                <p ALIGN="JUSTIFY">Standing Order 51,<i> as amended
                 (amendment No&nbsp;3A), agreed to.</p>
-                </i><b>
+                </i>
                 <p ALIGN="JUSTIFY">Standing Order 52 (Business Committee)</p>
+		<b>
                 <p ALIGN="JUSTIFY">The Initial Presiding Officer:</p>
                 </b>
                 <p ALIGN="JUSTIFY">Amendment No&nbsp;86: moved or not moved.</p>
@@ -5006,10 +5004,10 @@
                 <p ALIGN="JUSTIFY">Question accordingly negatived.</p>
                 <p ALIGN="JUSTIFY">Standing Order 52, as amended (amendments Nos&nbsp;3B
                 and 86), agreed to.</p>
-                </i><b>
+                </i>
                 <p ALIGN="JUSTIFY">Standing Order 53 (Conformity with Equality
                 Requirements — Special Committee On)</p>
-                </b><i>
+                <i>
                 <p ALIGN="JUSTIFY">Amendment (No 82) propsed: </i>Leave out</p>
                 </font><font SIZE="1">
                 <p ALIGN="JUSTIFY">&quot;CONFORMITY WITH EQUALITY REQUIREMENTS
@@ -5103,8 +5101,9 @@
                 matter of hunger and partly one of obviating bedsores. <i>[Laughter]</p>
                 <p ALIGN="JUSTIFY">Standing Order 53, as amended (amendments Nos&nbsp;3C,
                 81, 80 and 79), agreed to.</p>
-                </i><b>
+                </i>
                 <p ALIGN="JUSTIFY">Standing Order 54 (Public Accounts Committee)</p>
+		<b>
                 <p ALIGN="JUSTIFY">The Initial Presiding Officer:</p>
                 </b>
                 <p ALIGN="JUSTIFY">Amendment No&nbsp;78: moved or not moved?</p>
@@ -5171,9 +5170,10 @@
                 and negatived.</p>
                 <p ALIGN="JUSTIFY">Standing Order 54, as amended (amendment
                 No&nbsp;3D), agreed to.</p>
-                </i><b>
+                </i>
                 <p ALIGN="JUSTIFY">Standing Order 55 (Committee on Standards and
                 Privileges)</p>
+		<b>
                 <p ALIGN="JUSTIFY">The Initial Presiding Officer:</p>
                 </b>
                 <p ALIGN="JUSTIFY">Amendment 77: moved or not moved?</p>
@@ -5188,8 +5188,9 @@
                 <i>
                 <p ALIGN="JUSTIFY">Standing Order 55, as amended (amendments Nos&nbsp;3E
                 and 77), agreed to.</p>
-                </i><b>
+                </i>
                 <p ALIGN="JUSTIFY">Standing Order 56 (Audit Committee)</p>
+		<b>
                 <p ALIGN="JUSTIFY">The Initial Presiding Officer:</p>
                 </b>
                 <p ALIGN="JUSTIFY">Amendment No&nbsp;76: moved or not moved?</p>
@@ -5280,8 +5281,8 @@
                 wishing to speak on this group of amendments, and therefore we
                 shall proceed to the approval of the relevant Standing Orders
                 and amendments.</p>
-                <b>
                 <p ALIGN="JUSTIFY">Standing Order 57 (Order in the Assembly)</p>
+                <b>
                 <p ALIGN="JUSTIFY">The Initial Presiding Officer:</p>
                 </b>
                 <p ALIGN="JUSTIFY">The first amendment is No 75 standing in the
@@ -5309,20 +5310,20 @@
                 Buildings.&quot; —<i>&nbsp;[Mr P Robinson]</p>
                 </i></font><font SIZE="3"><i>
                 <p ALIGN="JUSTIFY">Standing Order 57, as amended, agreed to.</p>
-                </i><b>
+                </i>
                 <p ALIGN="JUSTIFY">Standing Order 58 (Visitors to the Assembly)</p>
-                </b><i>
+                <i>
                 <p ALIGN="JUSTIFY">Amendment (No&nbsp;71) made: </i>In paragraph
                 (4) after &quot;the&quot;, insert &quot;Public Gallery of
                 the&quot;. —&nbsp;[Mr&nbsp;P&nbsp;Robinson]</p>
                 <i>
                 <p ALIGN="JUSTIFY">Standing Order&nbsp;58, as amended, agreed
                 to.</p>
-                </i><b>
-                <p ALIGN="JUSTIFY">Standing Order 59 (Visitors to Committee) </b><i>agreed
+                </i>
+                <p ALIGN="JUSTIFY">Standing Order 59 (Visitors to Committee) <i>agreed
                 to.</p>
-                </i><b>
-                <p ALIGN="JUSTIFY">Standing Order 60 (Keeper of the House)</b> <i>agreed
+                </i>
+                <p ALIGN="JUSTIFY">Standing Order 60 (Keeper of the House) <i>agreed
                 to.</p>
                 <b>
                 <p ALIGN="center">Standards and Privilege</p>
@@ -5462,9 +5463,8 @@
                 <p ALIGN="JUSTIFY">I have received no requests to speak in
                 regard to this group of amendments, and therefore I will proceed
                 to decisions on them and on the Standing Orders in the group.</p>
-                <b>
                 <p ALIGN="JUSTIFY">Standing Order 61 (Members’ Interests)</p>
-                </b><i>
+                <i>
                 <p ALIGN="JUSTIFY">Amendment (No 70) made: </i>At the end of
                 paragraph (3) add</p>
                 </font><font SIZE="1">
@@ -5473,9 +5473,9 @@
                 —&nbsp;<i>[Mr P Robinson]</p>
                 </i></font><font SIZE="3"><i>
                 <p ALIGN="JUSTIFY">Standing Order 61, as amended, agreed to.</p>
-                </i><b>
+                </i>
                 <p ALIGN="JUSTIFY">Standing Order 62 (Privilege)</p>
-                </b><i>
+                <i>
                 <p ALIGN="JUSTIFY">Amendment (No 69) made: </i>In paragraph (1)
                 leave out &quot;three days&quot;. —&nbsp;<i>[Mr&nbsp;P&nbsp;Robinson]</p>
                 <p ALIGN="JUSTIFY">Amendment (No 67) made: </i>In paragraph (3)
@@ -6967,16 +6967,16 @@
                 Orders and amendments. We may have to ask the Secretary of State
                 for an extension beyond 10.00 pm if we are to cover this and
                 have the Adjournment debate.</p>
-                <b>
-                <p ALIGN="JUSTIFY">Standing Order 63 (Oath)</b> <i>agreed to.</p>
-                </i><b>
-                <p ALIGN="JUSTIFY">Standing Order 64 (Administration of Oath)</b>
+                <p ALIGN="JUSTIFY">Standing Order 63 (Oath) <i>agreed to.</p>
+                </i>
+                <p ALIGN="JUSTIFY">Standing Order 64 (Administration of Oath)
                 <i>agreed to.</p>
-                </i><b>
-                <p ALIGN="JUSTIFY">Standing Order 65 (Sub Judice) </b><i>agreed
+                </i>
+                <p ALIGN="JUSTIFY">Standing Order 65 (Sub Judice) <i>agreed
                 to.</p>
-                </i><b>
+                </i>
                 <p ALIGN="JUSTIFY">Standing Order 66 (News Media)</p>
+		<b>
                 <p ALIGN="JUSTIFY">The Initial Presiding Officer:</p>
                 </b>
                 <p ALIGN="JUSTIFY">Is amendment No 65 moved or not moved?</p>
@@ -6987,16 +6987,16 @@
                 Chairmen, not moved.</p>
                 <i>
                 <p ALIGN="JUSTIFY">Standing Order 66 agreed to.</p>
-                </i><b>
+                </i>
                 <p ALIGN="JUSTIFY">Standing Order 67 (Office of Clerk and
-                Records of the Assembly)</b> <i>agreed to.</p>
-                </i><b>
+                Records of the Assembly) <i>agreed to.</p>
+                </i>
                 <p ALIGN="JUSTIFY">Standing Order 68 (Remuneration and Pensions)
-                </b><i>agreed to.</p>
-                </i><b>
+                <i>agreed to.</p>
+                </i>
                 <p ALIGN="JUSTIFY">Standing Order 69 (Suspension of Standing
                 Orders)</p>
-                </b><i>
+                <i>
                 <p ALIGN="JUSTIFY">Amendment (No 64) proposed: </i>Leave out
                 Standing Order 69. —<i>&nbsp;[Mr P Robinson]</p>
                 <p ALIGN="JUSTIFY">8.45 pm</p>
@@ -7054,8 +7054,9 @@
                 is that Standing Order 69 is agreed to.</p>
                 <i>
                 <p ALIGN="JUSTIFY">Standing Order 69 agreed to.</p>
-                </i><b>
+                </i>
                 <p ALIGN="JUSTIFY">Standing Order 70 (Language)</p>
+		<b>
                 <p ALIGN="JUSTIFY">The Initial Presiding Officer:</p>
                 </b>
                 <p ALIGN="JUSTIFY">Is amendment 87 moved or not moved?</p>
@@ -7216,9 +7217,9 @@
                 <p ALIGN="JUSTIFY">Question accordingly negatived.</p>
                 <p ALIGN="JUSTIFY">9.15 pm</p>
                 <p ALIGN="JUSTIFY">Standing Order 70 agreed to.</p>
-                </i><b>
+                </i>
                 <p ALIGN="JUSTIFY">Standing Order 71 (Assembly Commission)</p>
-                </b><i>
+                <i>
                 <p ALIGN="JUSTIFY">Amendment (No 63) made:&nbsp;</i>Leave out
                 &quot;strength&quot; and insert &quot;strengths&quot;. — <i>[Mr
                 P Robinson]</p>
