--- /cygdrive/c/Documents and Settings/Matthew/My Documents/Projects/TheyWorkForYou/NI/parldata/cmpages/ni/ni2001-03-20.html	2006-11-12 10:27:07.687593600 +0000
+++ /cygdrive/c/Documents and Settings/Matthew/My Documents/Projects/TheyWorkForYou/NI/parldata/tmp/patchtmp-ni2001-03-20-pS13vb.html	2006-12-05 20:26:24.776059200 +0000
@@ -2217,7 +2217,7 @@
                 <p ALIGN="JUSTIFY">You did not say that about the First
                 Minister. <i>[Laughter].</p>
                 </i><b>
-                <p ALIGN="JUSTIFY">The Deputy First Minster:</p>
+                <p ALIGN="JUSTIFY">The Deputy First Minister:</p>
                 </b>
                 <p ALIGN="JUSTIFY">With regard to the second part of the Member’s
                 question, I will put it this way: there is no one in the
@@ -3565,7 +3565,7 @@
                 tuberculosis have resumed, as that was becoming quite a problem.
                 Apart from that there are no visits to farms at the moment.</p>
                 <b>
-                <p ALIGN="JUSTIFY">Mr Paisley Jr:</p>
+                <p ALIGN="JUSTIFY">Mr Paisley Jnr:</p>
                 </b>
                 <p ALIGN="JUSTIFY">I welcome the development of such farm
                 markets, but does the Minister agree that normalisation of trade
