--- /cygdrive/c/Documents and Settings/Matthew/My Documents/Projects/TheyWorkForYou/NI/parldata/cmpages/ni/ni2002-10-07.html	2006-11-12 10:27:53.343243200 +0000
+++ /cygdrive/c/Documents and Settings/Matthew/My Documents/Projects/TheyWorkForYou/NI/parldata/tmp/patchtmp-ni2002-10-07-PYRskp.html	2006-12-05 15:02:21.037336000 +0000
@@ -6287,7 +6287,7 @@
                 will give women more choice. I urge the Minister to instigate a
                 review of those pilot units as soon as possible.</p>
                 <b>
-                <p ALIGN="JUSTIFY">Ms Mc Williams:</p>
+                <p ALIGN="JUSTIFY">Ms McWilliams:</p>
                 </b>
                 <p ALIGN="JUSTIFY">I was not aware that a decision had been
                 taken on where the midwifery-led unit for the east of the
