--- /cygdrive/c/Documents and Settings/Matthew/My Documents/Projects/TheyWorkForYou/NI/parldata/cmpages/ni/ni2000-06-26.html	2006-11-12 10:26:28.661476800 +0000
+++ /cygdrive/c/Documents and Settings/Matthew/My Documents/Projects/TheyWorkForYou/NI/parldata/tmp/patchtmp-ni2000-06-26-9nQUzl.html	2006-12-05 19:05:19.460078400 +0000
@@ -3967,7 +3967,7 @@
               Dungannon from 31 July 2000.</p>
               <p ALIGN="JUSTIFY"><a name="AQO357/99">(AQO 357/99)</a></p>
               <b>
-              <p ALIGN="JUSTIFY">Ms de Brun:</p>
+              <p ALIGN="JUSTIFY">Ms de Brún:</p>
               </b>
               <p ALIGN="JUSTIFY">Níl socrú déanta go fóill i dtaca le
               géarsheirbhísí leighis d’othair-istigh in Otharlann
