--- /cygdrive/c/Documents and Settings/Matthew/My Documents/Projects/TheyWorkForYou/NI/parldata/cmpages/ni/ni2000-09-25.html	2006-11-12 10:26:37.784595200 +0000
+++ /cygdrive/c/Documents and Settings/Matthew/My Documents/Projects/TheyWorkForYou/NI/parldata/tmp/patchtmp-ni2000-09-25-2syv1K.html	2006-12-05 19:36:07.787843200 +0000
@@ -2100,8 +2100,8 @@
                 Birnie, to five minutes.</p>
                 <b>
                 <p ALIGN="JUSTIFY">The Deputy Chairperson of the Higher and
-                Further </b>Education, Training and Employment Committee
-                (Mr&nbsp;Carrick): The performance of the New Deal thus far can
+                Further Education, Training and Employment Committee
+                (Mr&nbsp;Carrick):</b> The performance of the New Deal thus far can
                 be measured in a number of ways. We could look at the
                 statistics; there are ample statistics in the KPMG report. There
                 has been plenty of press coverage, and we could make our
@@ -2713,7 +2713,7 @@
                 workers? Will they be subject to the same health and safety
                 regulations and equality of opportunity that apply to other
                 workers?</p>
-                <p ALIGN="JUSTIFY">These problems, a Cheann Comhairle<b>,</b>
+                <p ALIGN="JUSTIFY">These problems, a Cheann Comhairle,
                 are endemic in the short-term employment nature of the New Deal.
                 However, if New Deal is to make an impact on changing the
                 situation on the ground, it has to take into account local
@@ -5975,7 +5975,7 @@
                 <p ALIGN="JUSTIFY">5.15 pm</p>
                 </i><b>
                 <p ALIGN="JUSTIFY">The Minister of Agriculture and Rural
-                Development </b>(Ms Rodgers): I wish to place on record once
+                Development (Ms Rodgers):</b> I wish to place on record once
                 again that I welcome this report by the Committee of Agriculture
                 and Rural Development as part of its wider examination of
                 agricultural debt. I thank all Members for their comments and
