--- /cygdrive/c/Documents and Settings/Matthew/My Documents/Projects/TheyWorkForYou/NI/parldata/cmpages/ni/ni2000-02-07.html	2006-11-12 10:26:21.711483200 +0000
+++ /cygdrive/c/Documents and Settings/Matthew/My Documents/Projects/TheyWorkForYou/NI/parldata/tmp/patchtmp-ni2000-02-07-sDtW10.html	2006-12-05 18:53:55.907177600 +0000
@@ -1220,7 +1220,7 @@
                 everything that can be done to help the agriculture industry
                 will be done.</p>
                 </font><b>
-                <p ALIGN="JUSTIFY">Farming Industry Crisis</p>
+                <p ALIGN="center">Farming Industry Crisis</p>
                 </b><font SIZE="3">
                 <p ALIGN="JUSTIFY">4. <b>Mr McCarthy</b> asked the Minister of
                 Agriculture and Rural Development what approaches she has made
@@ -2487,7 +2487,7 @@
                 with research and development because, in today’s world, we
                 will not stand a chance of competing without it.</p>
                 </font><b>
-                <p ALIGN="JUSTIFY">Freedom of Information</p>
+                <p ALIGN="center">Freedom of Information</p>
                 </b><font SIZE="3">
                 <p ALIGN="JUSTIFY">4. <b>Mr Maskey</b> asked the First Minister
                 and the Deputy First Minister if there are any proposals to
@@ -2627,7 +2627,7 @@
                 </b>
                 <p ALIGN="JUSTIFY">The Member’s point is very well made.</p>
                 </font><b>
-                <p ALIGN="JUSTIFY">Civic Forum</p>
+                <p ALIGN="center">Civic Forum</p>
                 </b><font SIZE="3">
                 <p ALIGN="JUSTIFY">5. <b>Mr Ford</b> asked the First Minister
                 and the Deputy First Minister what criteria they will use to
@@ -2687,7 +2687,7 @@
                 capacity? Can they assure us that they will consider appointing
                 a representative of Families Acting for Innocent Relatives?</p>
                 <b>
-                <p ALIGN="JUSTIFY">The Deputy First Minster:</p>
+                <p ALIGN="JUSTIFY">The Deputy First Minister:</p>
                 </b>
                 <p ALIGN="JUSTIFY">No decision has been or will be made about
                 any sector of the Civic Forum until all the nominations have
