--- /cygdrive/c/Documents and Settings/Matthew/My Documents/Projects/TheyWorkForYou/NI/parldata/cmpages/ni/ni2001-01-22.html	2006-11-12 10:26:47.668808000 +0000
+++ /cygdrive/c/Documents and Settings/Matthew/My Documents/Projects/TheyWorkForYou/NI/parldata/tmp/patchtmp-ni2001-01-22-bz_0UI.html	2006-12-05 19:47:08.988603200 +0000
@@ -2584,7 +2584,7 @@
                 <p ALIGN="JUSTIFY">Clause 2, as amended, ordered to stand part
                 of the Bill.</p>
                 <b>
-                <p ALIGN="JUSTIFY">Clause 3 (Disturbing spawning beds, etc.)</p>
+                <p ALIGN="center">Clause 3 (Disturbing spawning beds, etc.)</p>
                 </b></i><b>
                 <p ALIGN="JUSTIFY">Mr Speaker:</p>
                 </b>
