--- /cygdrive/c/Documents and Settings/Matthew/My Documents/Projects/TheyWorkForYou/NI/parldata/cmpages/ni/ni2001-10-15.html	2006-11-12 10:27:16.680524800 +0000
+++ /cygdrive/c/Documents and Settings/Matthew/My Documents/Projects/TheyWorkForYou/NI/parldata/tmp/patchtmp-ni2001-10-15-Jc4IkX.html	2006-12-05 20:46:55.295459200 +0000
@@ -138,8 +138,8 @@
                 <p ALIGN="JUSTIFY">&nbsp;</p>
                 <font SIZE="3">
                 <p ALIGN="JUSTIFY">The Minister for Social Development (Mr
-                Morrow): I beg to move</p>
-                </font></b><font SIZE="3">
+                Morrow):</b> I beg to move</p>
+                </font><font SIZE="3">
                 <p ALIGN="JUSTIFY">That Standing Order 40(1) be suspended in
                 respect of the Final Stage of the Social Security Fraud Bill
                 (NIA 16/00).</p>
@@ -183,8 +183,8 @@
                 Final Stage</a></b></p>
                 <b><font SIZE="3">
                 <p ALIGN="JUSTIFY">The Minister for Social Development (Mr
-                Morrow): I beg to move</p>
-                </font></b><font SIZE="3">
+                Morrow):</b> I beg to move</p>
+                </font><font SIZE="3">
                 <p ALIGN="JUSTIFY">That the Social Security Fraud Bill (NIA
                 16/00) do now pass.</p>
                 <p ALIGN="JUSTIFY">This is a short, but important, Bill. Members
@@ -2353,7 +2353,7 @@
                 ensure that there are sufficient places in their respective
                 areas to meet potential demand.</p>
                 <b>
-                <p ALIGN="JUSTIFY">Ms Ramsay:</p>
+                <p ALIGN="JUSTIFY">Ms Ramsey:</p>
                 </b>
                 <p ALIGN="JUSTIFY">What is being done specifically about
                 bullying in schools and pupils who are afraid to attend?</p>
