--- /cygdrive/c/Documents and Settings/Matthew/My Documents/Projects/TheyWorkForYou/NI/parldata/cmpages/ni/ni2002-10-14.html	2006-11-12 10:27:51.871126400 +0000
+++ /cygdrive/c/Documents and Settings/Matthew/My Documents/Projects/TheyWorkForYou/NI/parldata/tmp/patchtmp-ni2002-10-14-m_WOID.html	2006-12-05 15:03:15.535700800 +0000
@@ -3327,8 +3327,8 @@
                 <b>
                 <p ALIGN="JUSTIFY">Mr G Kelly:</p>
                 </b>
-                <p ALIGN="JUSTIFY">Although I accept the general thrust of <b>‘</b>Developing
-                Better Services: Modernising<b> </b>Hospitals and Reforming
+                <p ALIGN="JUSTIFY">Although I accept the general thrust of ‘Developing
+                Better Services: Modernising Hospitals and Reforming
                 Structures’, issues such as the Mater Hospital must be sorted
                 out. Given the needs of the local community and the standards
                 that the hospital sets, its proposed services are insufficient.</p>
