--- /cygdrive/c/Documents and Settings/Matthew/My Documents/Projects/TheyWorkForYou/NI/parldata/cmpages/ni/ni2002-03-11.html	2006-11-12 10:27:38.501902400 +0000
+++ /cygdrive/c/Documents and Settings/Matthew/My Documents/Projects/TheyWorkForYou/NI/parldata/tmp/patchtmp-ni2002-03-11--zZHRH.html	2006-12-05 21:39:24.283481600 +0000
@@ -178,7 +178,7 @@
               Committee for Health, Social Services and Public Safety</b></a></p>
               </center>
               <p ALIGN="left"><font SIZE="3"><a href="#top">TOP</a></font></p>
-              <p ALIGN="left"><i><font SIZE="3"><b>Resolved:</b></p>
+              <p ALIGN="left"><i><font SIZE="3">Resolved:</p>
               </font></i><center><font SIZE="1">
               <p ALIGN="JUSTIFY">That Mr Tom Hamilton shall replace Mr Alan
               McFarland as a member of the Committee for Health, Social Services
@@ -2321,6 +2321,7 @@
               pupil information packs, and fact sheets covering each vocation<b>al
               area will be widely distributed. Information will also be
               available on the CCEA web site.</p>
+		<b>
               <p ALIGN="JUSTIFY">Mr K Robinson:</p>
               </b>
               <p ALIGN="JUSTIFY">The leading examination boards — Oxford
@@ -2517,6 +2518,7 @@
               teachers’ side seems to be a reasonable way forward, but I will
               await the views of the employing authorities before<b> making any
               final decision.</p>
+		<b>
               <p ALIGN="JUSTIFY">Mr Hamilton:</p>
               </b>
               <p ALIGN="JUSTIFY">The Minister informed the Committee for
