--- /cygdrive/c/Documents and Settings/Matthew/My Documents/Projects/TheyWorkForYou/NI/parldata/cmpages/ni/ni2002-01-21.html	2006-11-12 10:27:33.474673600 +0000
+++ /cygdrive/c/Documents and Settings/Matthew/My Documents/Projects/TheyWorkForYou/NI/parldata/tmp/patchtmp-ni2002-01-21-u5WjUz.html	2006-12-05 13:48:20.171684800 +0000
@@ -2408,7 +2408,7 @@
           recently published consultation document, ‘Promoting Sport in the
           Community,’ asks community amateur sports clubs to consider.</p>
           <i>
-          <p ALIGN="JUSTIFY">(Mr Deputy Speaker [Mr Mc Clelland] in the Chair)</p>
+          <p ALIGN="JUSTIFY">(Mr Deputy Speaker [Mr McClelland] in the Chair)</p>
           </i><b>
           <p ALIGN="JUSTIFY">Mr Carrick:</p>
           </b>
@@ -3473,7 +3473,7 @@
           <p ALIGN="JUSTIFY"><font SIZE="3"><a href="#top">TOP</a></font>
           </p>
           <b>
-          <p ALIGN="JUSTIFY">Outreach Tourism</p>
+          <p ALIGN="center">Outreach Tourism</p>
           </b>
           <font SIZE="3">
           <p ALIGN="JUSTIFY">6.</p>
