--- /cygdrive/c/Documents and Settings/Matthew/My Documents/Projects/TheyWorkForYou/NI/parldata/cmpages/ni/ni2006-11-27.html	2006-12-05 20:56:45.814584000 +0000
+++ /cygdrive/c/Documents and Settings/Matthew/My Documents/Projects/TheyWorkForYou/NI/parldata/tmp/patchtmp-ni2006-11-27-BNsp2h.html	2006-12-09 09:46:23.499489600 +0000
@@ -67,8 +67,7 @@
 <!-- #BeginEditable "data" -->
 <table align="center" width="640">
 	  <tr>
-		  <td><p class="W4SectionHeading"><span style="font-size:16.0pt; ">the transitional<br>
-		  </span><span style="font-size:16.0pt; ">assembly</span></p>
+		  <td><p class="W4SectionHeading"><span style="font-size:16.0pt; ">the transitional assembly</span></p>
 		  	<p class="H2DocumentDate" align="center" style="text-align:center;">Monday 27  November 2006</p>
 		  	<p class="B3BodyTextItalic">The sitting begun and suspended on Friday 24 November  2006 was resumed at 10.30 am (Madam Speaker in the Chair).</p>
 		  	<p class="H3SectionHeading">Assembly Business</p>
@@ -77,7 +76,7 @@
 		  	<p class="B3BodyText">Members will be aware that, following the incident that  occurred at the front entrance to Parliament Buildings on Friday morning, an  individual has been charged. The matter is, therefore, sub judice, so Members  will understand that I am constrained in what I can say about the incident at  this stage.</p>
 		  	<p class="B3BodyText">Having been briefed on that investigation by the Chief Constable  and an assistant chief constable, I can <span style="letter-spacing:-.2pt; ">say  that no one should underestimate the very real danger</span> that everyone in  the Building faced on Friday morning. The devices that were defused may have  been crude in nature, but they were no less life-threatening for that. <span style="letter-spacing:-.15pt; ">Moreover, we should not underestimate the  extraordinary</span> courage that was shown by our doorkeepers in confronting  that danger.</p>
 		  	<p class="B1SpeakersName"><strong>Some Members</strong>:  Hear, hear.</p>
-		  	<p class="B3BodyText"><strong><span style="letter-spacing:-.1pt; ">Madam Speaker:</span></strong> Our unarmed civilian doorkeepers  confronted an assailant whom they believed to be armed with a gun and  explosives. They disarmed and detained him. By doing so, they undoubtedly  prevented serious injury and possible loss of life. They selflessly endangered  their own lives in order to protect ours, and we are indebted to them for that.  We must not underestimate their bravery, nor understate our appreciation. Two  of the doorkeepers involved had to be taken to hospital, but I am very pleased  to report that neither suffered serious physical injury.</p>
+		  	<p class="B1SpeakersName"><strong>Madam Speaker</strong>: Our unarmed civilian doorkeepers  confronted an assailant whom they believed to be armed with a gun and  explosives. They disarmed and detained him. By doing so, they undoubtedly  prevented serious injury and possible loss of life. They selflessly endangered  their own lives in order to protect ours, and we are indebted to them for that.  We must not underestimate their bravery, nor understate our appreciation. Two  of the doorkeepers involved had to be taken to hospital, but I am very pleased  to report that neither suffered serious physical injury.</p>
 		  	<p class="B3BodyText">This morning, I met with all those who were involved to  express my own appreciation of, and admiration for, their actions and their  bravery. There will be a further  opportunity, Members, for the House to recognise  that bravery at a later date. Over the weekend, I spoke with party  leaders and many Members, and I <span style="letter-spacing:-.05pt; ">know that I  speak for the whole House &mdash; its Members</span>, the secretariat and Members&rsquo;  staff &mdash; and for the many members of the public who were present in the Building  for the sitting when I express our sincere gratitude and admiration for the  professionalism and courage that the doorkeepers displayed.</p>
 		  	<p class="B1SpeakersName"><strong>Some Members</strong>:  Hear, hear.</p>
 		  	<p class="B1SpeakersName"><strong>Madam Speaker</strong>:  In the light of Friday&rsquo;s events, questions have, quite reasonably, been asked  about the level of security in and around Parliament Buildings. We must be  mindful of Members&rsquo; own desire that Parliament Buildings be a public building  that is accessible and open to all our citizens and visitors. Nevertheless, the  Building must also be a place of safety where democracy can be exercised  without fear.</p>
@@ -115,7 +114,7 @@
 		  	<p class="B3BodyText">It is an illusion that the DUP is in control of the process  and can pull out at any time. The further it moves, inch by inch, towards the  Government&rsquo;s objective, the further it recedes the possibility of withdrawal.  It is equally foolish to assume that Sinn F&eacute;in will never deliver on the Pledge  of Office. A suitable form of words may be crafted for them, just as it has  been for the DUP.</p>
 		  	<p class="B3BodyText">Was it for enforced coalition with Sinn F&eacute;in that the men  and women of the security forces and the unionist community suffered and died?  Will we honour their memories by agreeing to share power with those who  approved the murders of Patsy Gillespie and Mr Hegarty, and who organised  Bloody Friday, Teebane, La Mon, the Droppin&rsquo; Well, the Shankill fish shop  bombing and countless other atrocities? That is the real question for those in  the DUP who want to move on by selling  their unionist principles for a mess of ministerial pottage. The DUP  leadership may be prepared to yield to threats to dissolve the Assembly, but  there are those who will never submit to such threats or be bought with  salaries, office, honours or patronage.</p>
 		  	<p class="B3BodyText">There is an historical precedent: Marshal P&eacute;tain became the  First Minister of Vichy France at the cost of his reputation and the people&rsquo;s  trust. There is still time for Ian Paisley to avoid a similar fate.</p>
-		  	<p class="B3BodyText"><strong>Madam Speaker: </strong>Mr McCartney, your time is up. Thank  you.</p>
+		  	<p class="B1SpeakersName"><strong>Madam Speaker</strong>: Mr McCartney, your time is up. Thank  you.</p>
 		  	<p class="B3BodyText">Before I call Mr Nesbitt to make his point of order, I  remind Members that a point of order is not an opport-unity for debate. It  would assist the House if Members would refer to the relevant Standing Order  when they raise a point of order. I shall not accept spurious points of order.</p>
 		  	<p class="B1SpeakersName"><strong>Mr Nesbitt</strong>:  Thank you, Madam Speaker. As a result of Friday&rsquo;s meeting, I wish to make two  points of order.</p>
 		  	<p class="B3BodyText">You referred this morning to Friday&rsquo;s meeting, which you  said was required under the Northern Ireland (St Andrews Agreement) Act 2006.  You also added that you, as Speaker, were not &ldquo;unfettered&rdquo;. I wish to address  that point.</p>
@@ -150,4 +149,4 @@
 	  <li><a href="http://search2.nics.gov.uk/kbroker/niassembly-transitional/search.jsp">Search</a></li>
 	</ul>
 </div></body>
-<!-- #EndTemplate --></html>
\ No newline at end of file
+<!-- #EndTemplate --></html>
