--- /cygdrive/c/Documents and Settings/Matthew/My Documents/Projects/TheyWorkForYou/NI/parldata/cmpages/ni/ni2002-09-16.html	2006-11-12 10:27:57.739564800 +0000
+++ /cygdrive/c/Documents and Settings/Matthew/My Documents/Projects/TheyWorkForYou/NI/parldata/tmp/patchtmp-ni2002-09-16-HlU9FN.html	2006-12-05 21:44:52.976118400 +0000
@@ -2411,6 +2411,7 @@
                 review body on post-primary education;<b> i</b>t will also need
                 to take account of the recommendations of <b>the Committee for
                 the Environment’s report on home-to-school transport.</p>
+		<b>
                 <p ALIGN="JUSTIFY">Mr Hussey:</p>
                 </b>
                 <p ALIGN="JUSTIFY">I am concerned, as is Ms McWilliams, that we
@@ -4937,28 +4938,28 @@
                 propose, by leave of the House, to put the Questions on these
                 motions en bloc.</p>
                 </font><b>
-                <p ALIGN="JUSTIFY">Business Committee</p>
+                <p ALIGN="center">Business Committee</p>
                 </b><i><font SIZE="3">
                 <p ALIGN="JUSTIFY">Resolved:</p>
                 </font></i><font SIZE="1">
                 <p ALIGN="JUSTIFY">That Ms Patricia Lewsley shall replace Mr
                 John Tierney on the Business Committee. <i>— [Dr Hendron.]</p>
                 </i></font><b>
-                <p ALIGN="JUSTIFY">Committee for the Environment</p>
+                <p ALIGN="center">Committee for the Environment</p>
                 </b><i><font SIZE="3">
                 <p ALIGN="JUSTIFY">Resolved:</p>
                 </font></i><font SIZE="1">
                 <p ALIGN="JUSTIFY">That Mr Michael Coyle shall serve on the
                 Committee for the Environment. <i>— [Dr Hendron.]</p>
                 </i></font><b>
-                <p ALIGN="JUSTIFY">Committee for Employment and Learning</p>
+                <p ALIGN="center">Committee for Employment and Learning</p>
                 </b><i><font SIZE="3">
                 <p ALIGN="JUSTIFY">Resolved:</p>
                 </font></i><font SIZE="1">
                 <p ALIGN="JUSTIFY">That Mr Michael Coyle shall serve on the
                 Committee for Employment and Learning. <i>— [Dr Hendron.]</p>
                 </i></font><b>
-                <p ALIGN="JUSTIFY">Committee on Standards and Privileges</p>
+                <p ALIGN="center">Committee on Standards and Privileges</p>
                 </b><i><font SIZE="3">
                 <p ALIGN="JUSTIFY">Resolved:</p>
                 </font></i><font SIZE="1">
