--- /cygdrive/c/Documents and Settings/Matthew/My Documents/Projects/TheyWorkForYou/NI/parldata/cmpages/ni/ni2002-09-24.html	2006-11-12 10:27:55.346123200 +0000
+++ /cygdrive/c/Documents and Settings/Matthew/My Documents/Projects/TheyWorkForYou/NI/parldata/tmp/patchtmp-ni2002-09-24-wUNesw.html	2006-12-05 15:01:43.082760000 +0000
@@ -525,7 +525,7 @@
                 Members have received, and Ministers will provide further
                 details in due course.</p>
                 <p ALIGN="JUSTIFY">The proposed allocation to the Department of
-                Agriculture and Rural Development<b> </b>represents an increase
+                Agriculture and Rural Development represents an increase
                 of 13% for 2003-04 on the allocations for 2002-03. The Executive
                 decided to allocate amounts from the innovation and
                 modernisation Executive programme fund, rising from £6 million
@@ -541,8 +541,8 @@
                 effectively. The proposals would also provide the Department
                 with a more realistic baseline for its animal health
                 responsibilities.</p>
-                <p ALIGN="JUSTIFY">The Department of Culture, Arts and Leisure<b>
-                </b>would receive<b> </b>£96·6 million in 2003-04 — an
+                <p ALIGN="JUSTIFY">The Department of Culture, Arts and Leisure
+                would receive £96·6 million in 2003-04 — an
                 increase of 9·9%. The draft Budget would enable that Department
                 to develop several key initiatives, including the implementation
                 of its sports strategy, which is important to many in the
@@ -550,7 +550,7 @@
                 grounds, measures to increase participation, and a new soccer
                 strategy aimed at developing the game in Northern Ireland.</p>
                 <p ALIGN="JUSTIFY">The proposed allocations for the Department
-                of Education<b> </b>would see<i> </i>expenditure rise to
+                of Education >would see expenditure rise to
                 £1528·6 million in 2003-04, an increase of 6·6% on the
                 allocations for 2002-03. That figure includes allocations
                 announced previously from the Executive programme funds and the
@@ -587,7 +587,7 @@
                 and numeracy problems as identified in the work of the
                 employability task force.</p>
                 <p ALIGN="JUSTIFY">The Minister of Enterprise, Trade and
-                Investment<b> </b>suggested a radical approach to the planning
+                Investment suggested a radical approach to the planning
                 of resources for his Department, which we have adopted to
                 reflect the reality that it is difficult to predict in advance
                 its needs and opportunities for industrial development support.
@@ -601,7 +601,7 @@
                 and will be, available if projects emerge for which financial
                 assistance would be effective and beneficial, while removing the
                 need to tie up resources that the Department of Enterprise,
-                Trade and Investment<b> </b>may not ultimately require – that
+                Trade and Investment may not ultimately require – that
                 would have limited the scope to allocate adequate budgets to
                 other priorities.</p>
                 <p ALIGN="JUSTIFY">The range between the minimum and maximum
@@ -610,7 +610,7 @@
                 is higher than the budget for 2002-03, and is probably the
                 maximum that could be required in the present economic climate.
                 This commitment would be underpinned by the Department of
-                Enterprise, Trade and Investment<b> </b>having a guaranteed
+                Enterprise, Trade and Investment having a guaranteed
                 first call on funding in-year up to the high end of the range,
                 to support investment projects offering value for money that
                 would not be otherwise affordable within the minimum Invest
@@ -695,7 +695,7 @@
                 changes in acute hospitals, primary care, the board/trust
                 structures and health promotion.</p>
                 <p ALIGN="JUSTIFY">The draft Budget would provide an increase of
-                10% in 2003-04 for the Department of<b> </b>the Environment
+                10% in 2003-04 for the Department of the Environment
                 above the 2002-03 allocation. That would enable the Department
                 of the Environment to advance work on the implementation of EU
                 Directives, including those on waste management, thus reducing
@@ -757,7 +757,7 @@
                 public sector that is already proceeding under the reinvestment
                 and reform initiative.</p>
                 <p ALIGN="JUSTIFY">For the Office of the First Minister and the
-                Deputy First Minister<b>, </b>the draft Budget proposes
+                Deputy First Minister, the draft Budget proposes
                 departmental expenditure totalling £38·6 million in 2003-04,
                 rising to £43·3 million in 2005-06. That would fund the review
                 of public administration and enable the establishment of the
@@ -5107,7 +5107,7 @@
               business over the coming months.</p>
               <b>
               <p ALIGN="JUSTIFY">The Junior Minister (Office of the First
-              Minister and the Deputy First Minister) Mr Haughey:</p>
+              Minister and the Deputy First Minister) (Mr Haughey):</p>
               </b>
               <p ALIGN="JUSTIFY">The debate in the Assembly begins the period of
               scrutiny of the Programme for Government. We have made a
